### DISCUSSION TITLE
How to install aireplay-ng ?

### DISCUSSION THREAD

**Discussion by wgf4242** [Homebrew/discussions#2791] [Everyday usage]
I installed aircrack by `brew install aircrack-ng`, then run aireplay-ng got
`-bash: aireplay-ng: command not found`

**Comment by gromgit** [comment]
If you want to know which binaries are installed by a specific formula, `brew ls <formula>` is your friend.

`brew ls aircrack-ng` tells me that `aireplay-ng` is installed in the `sbin` subdirectory, not the typical `bin`. This is probably because you'll need root to run it (not surprising, given what it's meant to do).

So you're looking at:
```
sudo ${HOMEBREW_PREFIX}/sbin/aireplay-ng ...
```
